diff --git a/dist/models/MonthDetail.js b/dist/models/MonthDetail.js
index e78d7a34b6f8cc5cd4f85517e3fdc21953cef80f..b59dc056d58abb164c0824bb5d077eb8c6dd7f96 100644
--- a/dist/models/MonthDetail.js
+++ b/dist/models/MonthDetail.js
@@ -48,7 +48,7 @@ function MonthDetailFromJSONTyped(json, ignoreDiscriminator) {
         'to_be_budgeted': json['to_be_budgeted'],
         'age_of_money': json['age_of_money'] == null ? undefined : json['age_of_money'],
         'deleted': json['deleted'],
-        'categories': (json['categories'].map(Category_1.CategoryFromJSON)),
+        'categories': (json['categories'] == null ? [] : json['categories'].map(Category_1.CategoryFromJSON)),
     };
 }
 exports.MonthDetailFromJSONTyped = MonthDetailFromJSONTyped;
@@ -69,7 +69,7 @@ function MonthDetailToJSONTyped(value, ignoreDiscriminator = false) {
         'to_be_budgeted': value['to_be_budgeted'],
         'age_of_money': value['age_of_money'],
         'deleted': value['deleted'],
-        'categories': (value['categories'].map(Category_1.CategoryToJSON)),
+        'categories': (value['categories'] == null ? [] : value['categories'].map(Category_1.CategoryToJSON)),
     };
 }
 exports.MonthDetailToJSONTyped = MonthDetailToJSONTyped;
